openapi/openapi.yml

```
@@ -121,62 +119,428 @@ components:
+                      * `RUN` - Rol Único Nacional
+                  # TODO: validate these options with Vector Capital
+                  enum:
+                    - RUT
+                    - RUN
+                gender:
+                  type: string
+                  description: >
+                    The user's gender.
+                      * `M` - Male
+                      * `F` - Female
+                  enum:
+                    - M
+                    - F
+                date_of_birth:
+                  type: string
+                  description: The user's date of birth.
+                  format: DD-MM-YYYY
+                email:
+                  type: string
+                  description: The user's email address.
+                  maxLength: 50
+                phone:
+                  type: string
+                  description: The user's phone number.
+                  maxLength: 50
+                martial_status:
+                  type: string
+                  description: >
+                    The user's marital status.
+                      * `M` - Married
+                      * `S` - Single
+                      * `D` - Divorced
+                      * `W` - Widowed
+                  # TODO: validate these options with Vector Capital
+                  enum:
+                    - M
+                    - S
+                    - D
+                    - W
+                profession:
+                  type: string
+                  description: The user's paid occupation.
+                  maxLength: 20
+                title:
+                  type: string
+                  description: The user's professional title.
+                  maxLength: 20
+                employer:
+                  type: string
+                  description: The user's employer.
+                  maxLength: 50
+                tax_payer_type:
+                  type: string
+                  description: The user's tax payment classification.
+                  # TODO: validate these options with Vector Capital
+                  enum:
+                    - Employee
+                    - Contractor
+                    - Business Owner
+                address_type:
+                  type: string
+                  description: The user's type of address.
+                  enum:
+                    - Home
+                    - Business
+                city:
+                  type: string
+                  description: The city of the user's address.
+                  maxLength: 50
+                zip:
+                  type: string
+                  description: The zip code of the user's address.
+                  maxLength: 7
+                street_address:
+                  type: string
+                  description: The user's street address.
+                  maxLength: 150
+                legal_address:
+                  type: string
```

```suggestion
                  type: boolean
```

Applies for all below